PypeS/pype.py

```diff
@@ -236,20 +240,21 @@ def Execute(self):
         for scriptNameAndArguments in self.ScriptList:
             self.PrintLog('')
             scriptName = scriptNameAndArguments[0]
-            moduleName = scriptName
-            scriptArguments = scriptNameAndArguments[1]
-            imported = True
             try:
-                exec('import '+ moduleName)
-            except ImportError, e:
-                self.PrintError(e.message)
+                module = importlib.import_module('vmtk.'+scriptName)
+                # Find the principle class to instantiate the requested action defined inside the requested writerModule script.
+                # Returns a single member list (containing the principle class name) which satisfies the following criteria:
+                #   1) is a class defined within the script
+                #   2) the class contains an attribute named 'self.SetScriptName'
+                # This makes the assumption that only one class per vmtkscript file contains the self.SetScriptName attribute, and that
+                # the class containing this attribute is the primary class to instantiate a new call to the scripts functionality
+                scriptObjectClasses = [x for x in dir(module) if isclass(getattr(module, x)) and hasattr(getattr(module, x), 'SetScriptName')]
```

Comment:
Hey Luca, 

I actually just pushed a fix for this to check for `issubclass(getattr(module, x), pypes.pypeScript)`. 

The reason I didn't do this in the first place was due to the same circular import error I was having  in the vmtkscripts module. This time it was the `pype.py` script importing 'pypes', which in turn imported `pype`... I didn't have a solution to that issue when I attempted to write this the first time, so just decided to check for `SetScriptName`. 

Now that i've got the imports all tightened up, the subclass solution works as expected.